src/dataexplorer/dataexplorer-application/src/types/IPatient.ts

```diff
@@ -3,18 +3,44 @@ export interface IPatient {
     name: IName[];
     birthDate: string;
     gender: string;
+    map(data: any): any;
+    address: IAddress[];
+    telecom: ITelecom[];
+    photo: IPhoto[];
+    url: string;
 }
 
-export interface IName {
-    family: string;
-    given: string[];
+export interface IState {
+    setName: (name: string) => void;
+    setSchema: (id: string) => void;
+    patientID: number;
+}
+export interface IRecord {
+    id: number;
+    schemaName: string;
 }
-
 export interface IPatientIdentifier {
     entry: IPatientResource[];
     total: number;
 }
 
+export interface IName {
+    family: string;
+    given: string[];
+}
+
 export interface IPatientResource {
     resource: IPatient;
 }
+export interface IAddress {
+    use: string;
+    line: string[];
+}
+
+export interface ITelecom {
+    value: number;
+}
+
+export interface IPhoto {
+    url: string;
+}
```

Comment:
Is it necessary to create an interface for an object, that only contains one field? I think it would be more readable to just say that the object is a string or a number.

Reply:
You're right. removed.